ocs_ci/ocs/bucket_utils.py

```diff
@@ -139,6 +140,27 @@ def retrieve_test_objects_to_pod(podobj, target_dir):
     return downloaded_objects
 
 
+def check_objects_in_bucket(
+    bucket_name, objects_list, mcg_obj, s3pod, tries=5, delay=10
+):
+    """
+    Checks object list present in bucket and compare it with uploaded object Lists
+    """
+    for i in range(tries):
+        obj_list = list_objects_from_bucket(
+            s3pod,
+            f"s3://{bucket_name}",
+            s3_obj=mcg_obj,
+        )
+        if objects_list == obj_list:
+            logger.info(f"Object list {obj_list}")
+            break
+        logger.info(f"Objects are not synced yet retrying after {delay} secs")
+        sleep(delay)
+    else:
+        raise UnexpectedBehaviour("Failed to sync objects")
```

Comment:
Please return a bool value and move the exception raising to the test that uses this utility function. 

The need for this has became apparent when I found myself needing to verify that some objects are *not* in a bucket.